test/addons/ocm-cluster/start

```diff
@@ -6,6 +6,7 @@
 import json
 import os
 import sys
+import yaml
```

Comment:
yaml is 3rd party library, so we separate it from standard library imports. See how it is imported in other files.